lib/interfaces/AuthenticationResponse.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * OpenID Connect Self-Issued ID Token Response.
+ */
+export default interface AuthenticationResponse {
+  /** MUST be set to 'https://self-issued.me'. */
+  iss: string;
```

Comment:
You can actually type this as a specific string to prevent other values from being used:
```suggestion
  iss: 'https://self-issued.me';
```

Reply:
Wouldn't that require the interface to be a class? Or does typescript allow literals in interfaces?